content/docs/examples/staticsite.md

```diff
@@ -156,10 +161,191 @@ cdev output hello_world_comp.staticsite.demofrontend.site_url
 
 Copy and paste the url into the browser to view the `Static Site`, `React` app.
 {{<tutorial_image>}}
-/images/react_example/react_site.png
+/images/staticsite_examples/react_example/react_site.png
+{{</tutorial_image>}}
+{{<break 2>}}
+
+{{<break 1>}}
+
+## Sync a Hugo Site to the Static Site Resource
+`Hugo` is a static website generator that converts content files, written in markdown, to static HTML and CSS pages. This example with show how Cdev integrates a Hugo site with the `Static Site` Resource.
+
+{{<tool_tip key="info" summary="Hugo Documentation">}}
+For more information about how to use `Hugo`, visit the **[official documentation](https:/https://gohugo.io/)**. 
+{{</tool_tip>}}
+
+### Requirements
+To generate a static site, you will need to have `Git` and `Hugo` installed. 
+{{<tool_tip key="info" summary="Hugo Installation Instructions">}}
+
+#### Git
+Instructions for installing `Git` can be found **[here](https://git-scm.com/downloads)**
+
+#### Hugo
+You can find the instructions for installing `Hugo` on your system **[here](https://gohugo.io/getting-started/installing/)**.
+{{</tool_tip>}}
+
+{{<break 1>}}
+
+The initial file structure should resemble the image below. If this is not your starting point for this example, consult the **[Getting Started](/docs/gettingstarted)** section to initialize a Cdev project before continuing.
```

Comment:
Again add the command to create a new project `cdev init` for the user to be able to copy